composer.json

```
@@ -60,7 +60,7 @@
 			"Google\\Task\\Composer::cleanup",
 			"Automattic\\WooCommerce\\GoogleListingsAndAds\\Util\\SymfonyPolyfillCleanup::remove",
 			"Automattic\\WooCommerce\\GoogleListingsAndAds\\Util\\GoogleAdsCleanupServices::remove",
-			"composer run-script remove-google-ads-api-version-support -- 18 19",
+			"composer run-script remove-google-ads-api-version-support -- 18 19 21",
```

@ankitrox it looks like the addition of `21` here can be removed now since you've reverted the changes to the AdsCampaign.php file that made this necessary (see [this conversation](https://github.com/woocommerce/google-listings-and-ads/pull/3053#discussion_r2273172725)).


```suggestion
			"composer run-script remove-google-ads-api-version-support -- 18 19",
```

---

Addressed [de3541f](https://github.com/woocommerce/google-listings-and-ads/pull/3053/commits/de3541fc338c592817a3074eaa9a18bf4040f304)

---

I was mistaken. It looks like v21 library files were still added even though none of those files were in use. I've reverted this change in [e1998c4](https://github.com/woocommerce/google-listings-and-ads/pull/3053/commits/e1998c487486eaa8f09fadfeb0901bc888c56c97) and confirmed that things are still working as intended.